frameworks/C++/poco/.gitignore

```diff
@@ -1 +0,0 @@
-/poco
```

Comment:
Just remove this file?

Reply:
this is what it looks like when a file is removed